tooling/miniutil/src/fmt/function.rs

```diff
@@ -159,20 +159,21 @@ fn fmt_terminator(t: Terminator, comptypes: &mut Vec<CompType>) -> String {
             let bb = fmt_bb_name(bb);
             format!("    goto -> {bb};")
         }
-        Terminator::If {
-            condition,
-            then_block,
-            else_block,
+        Terminator::Switch {
+            value,
+            cases,
+            fallback,
         } => {
-            let branch_expr = fmt_value_expr(condition, comptypes).to_string();
-            let then_bb = fmt_bb_name(then_block).to_string();
-            let else_bb = fmt_bb_name(else_block).to_string();
+            let branch_expr = fmt_value_expr(value, comptypes).to_string();
+            let mut case_strs = cases.iter()
+                                     .map(|(constant, successor)| format!("{}: {}", fmt_constant(constant).to_string(), fmt_bb_name(successor)))
+                                     .collect::<Vec<String>>();
+            case_strs.push(format!("otherwise: {}", fmt_bb_name(fallback)));
```

Comment:
Maybe we should spell this `default:` like in C?

Reply:
I based the formatting on how Rust represents MIR. It uses `otherwise`.

Reply:
Okay let's be consistent with MIR then.